SPECS-EXTENDED/ppp/ppp.spec

```diff
@@ -188,6 +188,9 @@ install -p %{SOURCE11} %{buildroot}%{_sysconfdir}/sysconfig/network-scripts/ifdo
 %doc PLUGINS
 
 %changelog
+* Wed Aug 30 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2.4.7-37
+- Add patch for CVE-2022-4603
```

Comment:
I think the automation only added the patch file; did not do a `Patch0: CVE-2022-4603.patch`.